lib/addons/argocd/index.ts

```diff
@@ -1,39 +1,42 @@
+import bcrypt = require('bcrypt');
 import { Construct } from "@aws-cdk/core";
 import { HelmChart, KubernetesManifest, ServiceAccount } from "@aws-cdk/aws-eks";
 import { ManagedPolicy } from "@aws-cdk/aws-iam";
-import { ApplicationRepository, ClusterAddOn, ClusterPostDeploy, ClusterInfo, Team } from "../../spi";
 
+import * as spi from "../../spi";
 import { getSecretValue } from '../../utils/secrets-manager-utils';
 import { sshRepoRef, userNameRepoRef } from './manifest-utils';
 import { btoa } from '../../utils/string-utils';
-import bcrypt = require('bcrypt');
 import { Constants } from "..";
 
-
 /**
- * Configuration options for ArgoCD add-on.
+ * Configuration options for add-on.
  */
 export interface ArgoCDAddOnProps {
-
     /**
      * Namespace where add-on will be deployed. 
      */
-    namespace?: string,
+    namespace?: string
+
+    /**
+    * Helm chart version to use to install.
```

Comment:
Since this is a cleanup PR, recommend adding `@default ....` for optional parameters to suggest what the default value is which will be displayed as hint in the IDE.